src/screens/Register.jsx

```diff
@@ -21,21 +32,53 @@ const Register = () => {
   };
 
   const assignUserRole = async (email, values) => {
-    const account = await assignRole({ email, ...values });
-    console.log(account);
+    if (role === "candidate") {
```

Comment:
Instead of duplicating code, you could just do:

assignUserRole() and provide it the role that you will need in the request.js, and format you object like so: 
{role: {email, ...values, etc}}